admin/plugins.php

```diff
@@ -261,8 +275,11 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	$action_links = apply_filters( 'plugin_install_action_links', $action_links, $plugin_data );
 
 	$last_updated_timestamp = strtotime( $plugin_data['last_updated'] );
+
+	$standalone_plugins_file_map = perflab_get_standalone_plugins_file_map();
+	$plugin_file                 = isset( $standalone_plugins_file_map[ $plugin_data['slug'] ] ) ? wp_create_nonce( 'perflab_activate_plugin_' . $standalone_plugins_file_map[ $plugin_data['slug'] ] ) : '';
 	?>
-	<div class="plugin-card plugin-card-<?php echo sanitize_html_class( $plugin_data['slug'] ); ?>">
+	<div class="plugin-card plugin-card-<?php echo sanitize_html_class( $plugin_data['slug'] ); ?>" data-module="<?php echo esc_attr( $plugin_data['slug'] ); ?>" data-plugin="<?php echo esc_attr( $plugin_file ); ?>">
```

Comment:
Two points of feedback here:
* The `data-module` isn't used as far as I can see.
* Couldn't we put these attributes on the Install link `a` tag? That would be closer to where it applies and would allow us to avoid unnecessary parent lookups in the jQuery code.

Reply:
I think the WP updates.js script expects `data-plugin` here and not on the link.

Reply:
Thanks for the feedback. The custom attribute is moved to href.

Reply:
I thought this was just a custom data attribute. @swissspidy What is the purpose of `data-plugin` on the card, and is that relevant here or not?

Reply:
Yes, that is a custom field that I use to retrieve the plugin slug in JS. However, I'm uncertain about how the JavaScript utilizes that particular field in the code located here: [link to the code on GitHub](https://github.com/WordPress/wordpress-develop/blob/trunk/src/js/_enqueues/wp/updates.js#L2167) 😞